packages_rs/nextclade/src/tree/tree.rs

```diff
@@ -35,8 +35,20 @@ impl TreeNodeAttr {
 }
 
 #[derive(Clone, Serialize, Deserialize, Validate, Debug)]
-pub struct TreeNodeAttrInt {
+pub struct TreeNodeAttrF64 {
   pub value: f64,
+
+  #[serde(flatten)]
+  pub other: serde_json::Value,
+}
+
+impl TreeNodeAttrF64 {
+  pub fn new(value: f64) -> Self {
+    Self {
+      value,
+      other: serde_json::Value::default(),
```

Comment:
What is the purpose of these `other`s in the `TreeNodeAttr`s? I don't think I've understood that yet

Reply:
I think that the object there can contain other fields (which we don't care about). This is to tell serde to potentially expect more fields and to gather them in this dict-like structure.

It's no different from the string-based counterpart of this type.

Anna found these fields useful for prototyping (you can add things to JSON without modifying the struct)

Reply:
What is the purpose of these `other`s in the `TreeNodeAttr`s? I don't think I've understood that yet

Reply:
I believe there can be attrs on the tree which look like:

```json
{
	"name": "Foo"
	"description": "Yep, that's a foo!"
    "value: "foo!"
}
```

When parsing, the JSON `value` will go to struct's `value`, while the JSON's `description` and `name` will go to the struct's `other` "dict".

It's not strictly necessary for the f64 variant, which we only use in a single place we ourselves invented, but I like to add it anyway, to avoid "unexpected extra field" errors in case this changes.